### ISSUE TITLE
ANN Model Results and Performance

### ISSUE THREAD

**Issue by athallah-naufal** [athallah-naufal/ML_GreenBuildingANN#1]
## 🧠 Model Performance: ANN

The table below summarizes the performance of the Artificial Neural Network (ANN) model on both the training and testing datasets, as well as the detailed performance for each variable (velocity, temperature, and relative humidity).

### Training vs Testing Results

<table>
  <thead>
    <tr>
      <th>Metric</th>
      <th>Training</th>
      <th>Testing</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>R²</strong></td>
      <td>0.955403</td>
      <td>0.862252</td>
    </tr>
    <tr>
      <td><strong>MSE</strong></td>
      <td>0.00189887</td>
      <td>0.00556519</td>
    </tr>
    <tr>
      <td><strong>MAE</strong></td>
      <td>0.0286858</td>
      <td>0.0414909</td>
    </tr>
  </tbody>
</table>

### Detailed Performance by Variable (Velocity, Temperature, Relative Humidity)

<table>
  <thead>
    <tr>
      <th>Variable</th>
      <th>R²</th>
      <th>MSE</th>
      <th>MAE</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>Velocity (v)</strong></td>
      <td>0.86823</td>
      <td>0.00869</td>
      <td>0.05261</td>
    </tr>
    <tr>
      <td><strong>Temperature (T)</strong></td>
      <td>0.85106</td>
      <td>0.00642</td>
      <td>0.02689</td>
    </tr>
    <tr>
      <td><strong>Relative Humidity (RH)</strong
></td>
      <td>0.95339</td>
      <td>0.00157</td>
      <td>0.04496</td>
    </tr>
  </tbody>
</table>

### 📉 Training and Validation Loss

The graph below shows the training and validation loss over epochs for the ANN model. This indicates how well the model converged during training and how it performed on the validation dataset.

![Loss Epoch](https://github.com/user-attachments/assets/fab9cfa8-0ab3-46db-86fc-265e1fdbb06c)

From the graph, we can see that the loss decreases rapidly during the early epochs and stabilizes as the model continues to train, with both the training and validation loss remaining low, indicating a good fit without overfitting.
